experimental/framework/fluid-static/src/types.ts

```diff
@@ -77,7 +77,7 @@ export interface ContainerSchema {
  * Event that triggers when the roster of members in the Fluid session change
  */
 export interface IServiceAudienceEvents extends IEvent {
-    (event: "membersChanged", listener: () => void): void;
+    (event: "membersChanged", listener: (members: Map<string, IMember>) => void): void;
```

Comment:
If `IServiceAudienceEvent` takes in the generic `M` from `IServiceAudience` then the event-listener can also provide strong typing for each client's member object
```suggestion
    (event: "membersChanged", listener: (members: Map<string, M>) => void): void;
```